etc/kayobe/environments/aufn-ceph/kolla/inventory/overcloud-services.j2

```diff
@@ -0,0 +1,561 @@
+# This inventory section provides a mapping of services to components.
```

Comment:
This might need an update for yoga. Here is the source: https://github.com/stackhpc/kayobe/blob/stackhpc/yoga/ansible/roles/kolla-ansible/templates/overcloud-services.j2